src/oscserver/oscserver.cpp

```diff
@@ -0,0 +1,170 @@
+#include "oscserver.h"
+
+namespace {
+    void oscErrorHandler(int err, const char* msg, const char* path) {
+        qWarning() << QString("%1. OSC path: %2. Error Code: %3.").arg(QString::fromLatin1(msg)).arg(QString::fromLatin1(path)).arg(err);
+    }
+
+    int oscMsgHandler(const char* path, const char* types, lo_arg** argv, int argc, void* data, void* userData) {
+        Q_UNUSED(userData);
+
+        QRegularExpression pathRegEx("\\/(.+)\\/(.+)");
+        QRegularExpressionMatch pathMatch = pathRegEx.match(QString::fromLatin1(path));
+
+        if (!pathMatch.hasMatch()) {
+            qWarning() << "Invalid OSC path: " << QString::fromLatin1(path);
+            qWarning() << "Proper OSC path format: /<group>/<control>";
```

Comment:
Here, we put the group into the root of the address field while we send out data with the Mixxx prefix.
Will we ever have a problem with this?

Reply:
I thought the Mixxx prefix would make sense for outgoing data because, if we're interacting with a client that is compatible with multiple OSC apps, the prefix clarifies to the client that they are receiving data from Mixxx rather than another application that may use paths like /Channel1/play (because those are pretty generic names). Incoming data, on the other hand, was explicitly addressed to Mixxx, so the prefix is unnecessary because, e.g., /Channel1/play, can only have one possible meaning in that context. Please let me know if that makes sense or if I'm overthinking it.

Reply:
Yes the mixxx prefix for the outgoing data makes sense. I think from this point of view the incoming messages should have also a mixxx prefix. 
In this case a transmitting button and a receiving LED can be on the same address.

Reply:
Sounds good. I will fix that.